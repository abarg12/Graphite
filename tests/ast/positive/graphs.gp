graph<int, tree, connected> sometraversal(graph<int, a, b, c> arg1) {
    graph<int, x, y, z> g;
    node<int> n1;
    g.addNode(n1);
    return g;
}

graph<int> myGraph;
node<int> n2;
myGraph.addNode(n2);
