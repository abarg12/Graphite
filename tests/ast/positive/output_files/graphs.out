graph <int> sometraversal(graph <int> arg1) 
{
graph <int> g;
node<int> n1;
g.addNode(n1);
return g;
}
graph <int> myGraph;
node<int> n2;
myGraph.addNode(n2);
